emcc.py

```diff
@@ -1768,7 +1769,7 @@ def get_eliminate():
       if shared.Settings.CYBERDWARF:
           execute([shared.PYTHON, shared.path_from_root('tools', 'emdebug_cd_merger.py'), target + '.cd', target+'.symbols'])
 
-      if options.debug_level >= 4:
+      if options.debug_level >= 4 and not shared.Settings.BINARYEN:
         emit_source_maps(target, optimizer.js_transform_tempfiles)
```

Comment:
since this method is just for js and not wasm, how about renaming it to `emit_js_source_maps`?